ui/modal/info_modal.go

```diff
@@ -136,6 +142,11 @@ func (in *InfoModal) Handle() {
 		in.DismissModal(in)
 		in.negativeButtonClicked()
 	}
+
+	if in.modal.BackdropClicked() {
```

Comment:
A `cancelable` property should be added for pages that do not want the modal to be canceled by tapping outside, some pages require a definite Yes or No.